### ISSUE TITLE
508-defect-3 [COGNITION]: Links SHOULD be styled as links

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18395]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

- As a user, I expect links to be styled as links so that I know I can open them in a new tab.
- As a voice AT user, I expect links to be styled as links so that I can provide an accurate command based on what I see.
- As a screen reader user with low vision, I expect the screen reader to announce what I see on the page.
- As a user, I expect links to be styled consistently as links so that I do not confuse them with buttons.

## Details

The "go to ___ form" link is styled as a `button` which is materially dishonest. This causes several issues:
- It's materially dishonest and damages habituation on VA.gov 
- Voice assistive tech users may say "Click go to xyz form _button_" which may result in failure as it is tagged as a `a`
- Screen reader users with low vision may become confused at a button being announced as a link
- Sighted users may not know that they can open this in a new tab because it looks like a button, especially on mobile which gives a stronger affordance for a tap vs. a tap/hold

This is a sitewide long-tail effort. [To better track button/link inconsistencies, please view this Mural.](https://app.mural.co/invitation/mural/vsa8243/1610052905994?sender=u2134c22982ad9c7b41798011&key=87284ca2-fd99-431d-b086-5890abf480ac).

## Acceptance Criteria

- [ ] Links are styled as links

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Solution (if known)

The easiest way to solve this is to style the link as a link. There are design drawbacks to this if the standard underline/blue text method is used as it may not be as visually prominent or action-nudging. That being said, we can be flexible/creative with how we approach this. For example, consider [NHS's action link](https://service-manual.nhs.uk/design-system/components/action-link) which manages to maintain the styling of a link while prompting action.

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)

## Screenshots or Trace Logs

<img width="802" alt="Screen Shot 2021-01-12 at 10 50 54 AM" src="https://user-images.githubusercontent.com/14154792/104339863-10614500-54c6-11eb-97b8-de1e058e7293.png">

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO We'll need some help from design to see what they would want to do different about the design of the search result since there is an a11y issue with the design of the button.

**Comment by MarciMcGuireGCIO** [comment]
Moving all outstanding Find a VA Form 508 issues into a single Epic to clean up backlog.

**Comment by MarciMcGuireGCIO** [comment]
Sprint planning note - this is included in the UI consistency work currently in progress.

**Comment by MarciMcGuireGCIO** [comment]
Note - we need to get Ryan together with Josh & Trevor on a solution for this.
